frontend/package.json

```diff
@@ -44,8 +46,11 @@
     "enzyme": "^3.11.0",
     "enzyme-adapter-react-16": "^1.15.2",
     "enzyme-to-json": "^3.5.0",
+    "jsonwebtoken": "^8.5.1",
     "prettier": "2.0.5",
+    "prop-types": "^15.7.2",
     "react-redux": "^7.2.1",
```

Comment:
@marcianszlovak probably react-redux is also needed in the dependencies, not just in the devDependencies